common/djangoapps/track/views/__init__.py

```diff
@@ -91,7 +91,6 @@ def user_track(request):
             pass
 
     context_override = contexts.course_context_from_url(page)
-    context_override['username'] = username
```

Comment:
Do we maybe want to add in `user.id` instead? Otherwise there is no traceablity

Reply:
good point